src/App.scss

```diff
@@ -11,6 +11,10 @@ a {
   }
 }
 
+svg {
```

Comment:
just a thought, but do we want to be more targeted with this rather than applying it to all svgs?

Reply:
I think we want this everywhere - this is just to stop svgs shrinking when there's not much space